ui/selectors/accounts.ts

```diff
@@ -28,11 +31,20 @@ export function isSelectedInternalAccountBtc(state: AccountsState) {
   return isBtcAccount(getSelectedInternalAccount(state));
 }
 
-export function hasCreatedBtcMainnetAccount(state: AccountsState) {
+function hasCreatedBtcAccount(
+  state: AccountsState,
+  isAddressCallback: (address: string) => boolean,
+) {
   const accounts = getInternalAccounts(state);
   return accounts.some((account) => {
-    // Since we might wanna support testnet accounts later, we do
-    // want to make this one very explicit and check for mainnet addresses!
-    return isBtcAccount(account) && isBtcMainnetAddress(account.address);
+    return isBtcAccount(account) && isAddressCallback(account.address);
   });
 }
+
+export function hasCreatedBtcMainnetAccount(state: AccountsState) {
+  return hasCreatedBtcAccount(state, isBtcMainnetAddress);
+}
+
+export function hasCreatedBtcTestnetAccount(state: AccountsState) {
+  return hasCreatedBtcAccount(state, isBtcTestnetAddress);
```

Comment:
This isBtcTestnetAddress would return true for a non-EVM and non-BTC address. We can probably address this later when we support other non-EVM chains

Reply:
Good point, and yes we would need to address this! Let's keep note of that for later